content/en/developers/metrics/_index.md

```diff
@@ -17,20 +18,28 @@ further_reading:
   text: "Official and Community created API and DogStatsD client libraries"
 ---
 
-A Datadog metric is defined by the properties below. Refer to the [Metrics Introduction documentation][1] to learn how to graph metrics within Datadog.
+## Overview
+
+If a metric is not submitted from one of the [350+ Datadog integrations][1] it's considered a custom metric (with [some exception][2]).
+
+A custom metric refers to **a unique combination of metric name, host, and tag values**. In general, any metric you send using [DogStatsD][3], or through a [custom Agent Check][4] is a custom metric. There are no enforced fixed rate limits on custom metric submission. If your default allotment is exceeded, you are billed according to [Datadog's billing policy for custom metrics][5].
```

Comment:
This paragraph feels disjoint - we're mentioning allocation, rate limits and billing along with admin rights in consecutive sentences. I'd rather have a section at the very bottom that calls that stuff out in a separate section as allocation/tracking to avoid interrupting a user's understanding of what a custom metric is and why it's valuable

Reply:
It's to quickly redirect people to the billing aspect of it to avoid confusion, i re-worded the sentence to make it more streamlined.